.env.production

```diff
@@ -6,3 +6,6 @@ CP_ANALYTICS_API=https://cp-analytics.reevoo.com/api/v1/
 # Tableau
 TABLEAU_GATEWAY_API=https://tableauapigateway.reevoo.com/api/v1/
 TABLEAU_HOST=https://tableau.reevoo.com/
+GTM_ENV=env-1
+GTM_AUTH_KEY=CmO5CrJ2SMIyrU8Ko9t6ag
```

Comment:
@osa911 I think production AuthKey shouldn't be stored in the repository

Reply:
Maybe you are right, but repo is private
And I think it's ok.
This is necessary mostly for debugging tracking and that's all
Main key is GTM_ID